torch/onnx/symbolic_opset13.py

```diff
@@ -312,3 +313,62 @@ def repeat_interleave(g, self, repeats, dim=None, output_size=None):
     loop_out = loop.node().output()
     loop_out = g.op("ConcatFromSequence", loop_out, axis_i=dim)
     return loop_out
+
+
+@parse_args("v", "i", "i", "i")
+def diagonal(g, self, offset, dim1, dim2):
+    dim1_size = size(g, self, dim=g.op("Constant", value_t=torch.LongTensor([dim1])))
+    dim2_size = size(g, self, dim=g.op("Constant", value_t=torch.LongTensor([dim2])))
+
+    # Create appropriate mask
+    mask_shape = [dim1_size, dim2_size]
+    mask_shape = g.op("Concat", *mask_shape, axis_i=0)
```

Comment:
Re-assigning a variable to one of a totally different type seems to me to harm readability.
Can we rewrite to:
`mask_shape = g.op("Concat", dim1_size, dim2_size, axis_i=0)`
?